packages/docusaurus/src/client/App.tsx

```diff
@@ -27,6 +41,7 @@ function App(): JSX.Element {
   return (
     <DocusaurusContext.Provider
       value={{siteConfig, siteMetadata, globalData, isClient}}>
+      <CSSLoadingWarningMessage isClient />
```

Comment:
```suggestion
      <CSSLoadingWarningMessage isClient={isClient} />
```

Reply:
Hi,
The pre-commit hook changed this from `isClient={isClient}` to `isClient`